### PR TITLE
Fix download incomplete query and bad behaviour after category selection

### PR DESCRIPTION
**Pull request by ArnaudBelcour**
Fix the incomplete query made when pressing the Download query button. Now VALUES and FILTER are contained in the query.

Also during request on AskOmics, when the user select attributes of category present in the right panel and deselect them and then select the empty value in the panel (to select all the value of this category), the request created was incorrect (it's displayed no result). 
This fix delete the empty attribute in the category selection panel and add a cross (at the left of the eye enabling category). When clicking on the cross it deselect all the previous attributes (allowing request on all attributes of the category).

### CODE REVIEW

**Comment by ofilangi** [comment]
+1

**Comment by ArnaudBelcour** [comment]
There was an error with cleaning filter for number. Clicking on the cross doesn't work. It has been fixed.

Also add a cross for cleaning entity's name filter in the choice panel "Name".

The cross icon is now placed at the right of the eye icon, like the actuals icons for node selection and suppression.

File: askomics/static/js/askomics.js [hunk 1]
```diff
@@ -160,6 +157,21 @@ function hideModal(){
 }
 
 
+function downloadTextAsFile(filename, text) {
+    // Download text as file
```
Comment (abretaud) [inline askomics/static/js/askomics.js]: Can you use 4 spaces indentation here?

Comment (ArnaudBelcour) [inline askomics/static/js/askomics.js]: Yes, i forgot them.
It has been fixed.

**Comment by abretaud** [comment]
Thank you!

**Pull request merged**